.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-  "python.pythonPath": "python3",
+  "python.pythonPath": "/Users/wujingwujing/miniconda3/envs/wechaty/bin/python",
```

Comment:
Please fix this path, or other developers will fail to load.

Reply:
Oh, it's my fault to commit this configuration section. We should keep this file in repo but ignored in `.gitignore` file.